package.json

```diff
@@ -121,6 +121,7 @@
     "istanbul-api": "2.1.6",
     "istanbul-reports": "3.0.2",
     "jasmine-core": "~3.5.0",
+    "jasmine-protractor-browser-log-reporter": "cf-stratos/jasmine-protractor-browser-log-reporter",
```

Comment:
`"protractor-console": "^3.0.0"` can be nuked now?

Reply:
Done